py_bind/test/test_trafo_bindings.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from optv.parameters import *
```

Comment:
Please don't use ``import *``. It invites namespace pollution, and fucks up autocompletion and stuff in IDEs.